packages/material-ui/src/Stepper/Stepper.test.js

```diff
@@ -201,30 +240,37 @@ describe('<Stepper />', () => {
   });
 
   it('renders with a null child', () => {
-    const wrapper = shallow(
+    // I'm not sure if this test brings any value
```

Comment:
It was added after a bug, crash on cloneElement. If we move to the context API, we will be able to remove it, but later :)

```suggestion
```